DAO.sol

```diff
@@ -821,7 +822,10 @@ contract DAO is DAOInterface, Token, TokenCreation {
 
 
     function halveMinQuorum() returns (bool _success) {
-        if (lastTimeMinQuorumMet < (now - quorumHalvingPeriod)) {
+        // this can only be called after `quorumHalvingPeriod` has passed or at anytime
+        // by the curator with a delay of at least `minProposalDebatePeriod` between the calls
+        if ((lastTimeMinQuorumMet < (now - quorumHalvingPeriod) || msg.sender == curator)
+			&& lastTimeMinQuorumMet < (now - minProposalDebatePeriod)) {
```

Comment:
This is only there to prevent the curator calling it hundreds of times within a minute, and then start an attack with low minQourum. On the other hand, if your curator is malicious, you should split anyway, and this gives you the time to do it.